src/dialogs/more-info/controls/more-info-weather.ts

```diff
@@ -187,6 +187,7 @@ class MoreInfoWeather extends LitElement {
                             this.hass.locale
                           )}
                         </div>
+                        <div class="templow">—</div>
```

Comment:
This is weird, for an integration that doesn't provide min temps, we now just don't show the column, what is good I think, now we will show a `-` on each row?

If we want to show a `-` we should only do it if one of the other rows has a min temp value.

Reply:
This is a bit of an edge case; in the evening Canada's weather service stops providing a high temperature for the current day, but still provides a low for the night ahead. This is what is looks like on their website:

<img width="664" alt="Screen Shot 2021-05-20 at 8 32 13 PM" src="https://user-images.githubusercontent.com/16008714/119064947-7c8b1280-b9aa-11eb-859f-1a78c5fa0990.png">

Reply:
I adjusted the logic now to check if we have at least one of `templow` or `temperature`, otherwise we hide the column (or row in the more-info) as before.